src/components/05_pages/Permissions/Permissions.js

```diff
@@ -1,17 +1,98 @@
-import React from 'react';
-import { css } from 'emotion';
+import React, { Fragment, Component } from 'react';
+import Loading from '../../Helpers/Loading';
 
-const styles = {
-  title: css`
-    text-decoration: underline;
-  `,
+const Permissions = class Permissions extends Component {
+  state = {
+    loaded: false,
+  };
+  componentDidMount() {
+    Promise.all([
```

Comment:
Also a general question. Don't we need some handling for the case that the component is not longer active but the promise resolves? (ideally we would cancel the promise as part of ```componentDidUnmount``` or something like that.

Reply:
Yes, we could set the promise up in the constructor and just execute when the component mounts, and cancel it later.

Reply:
Let's keep that up as a todo, I guess?

Reply:
I'll just get these wrapped up today. :)